src/gui.ml

```diff
@@ -154,11 +155,51 @@ let draw_position (bd : Game.t) (imgs : image list) =
   let bd = board_to_array bd in
   draw_position_rows bd imgs 0
 
-let draw_game (bd : Game.t) =
+let rec get_potential_squares (move_list : move list) currX currY :
```

Comment:
Make sure to document this function. It is super important to the whole functionality.

Reply:
Also, maybe put this in helper.ml?